libethereum/BlockChainSync.h

```diff
@@ -161,6 +161,7 @@ class BlockChainSync final: public HasInvariants
     u256 m_syncingTotalDifficulty;				///< Highest peer difficulty
 
     Logger m_logger{createLogger(VerbosityDebug, "sync")};
+    Logger m_loggerInfo{createLogger(VerbosityInfo, "sync")};
```

Comment:
Do you need to create a logger for every combination? Can't we apply the verbosity level at the place of `LOG` macro?

Reply:
The logger should be defined somewhere anyway (either global or local/member variable, the latter preferable)
But one alternative could be to have a logger without fixed verbosity, then a macro like `LOGV(m_logger, VerbosityInfo) << ...`
which we can make `LOG_INFO(m_logger) << ...`